dashboard/middleware/disableintrospection.py

```diff
@@ -0,0 +1,16 @@
+from graphql import GraphQLField, GraphQLObjectType, GraphQLSchema, GraphQLString
+
+
+class IntrospectionDisabledException(Exception):
+    pass
```

Comment:
I might consider adding some more context to this exception like this but maybe isn't necessary. I haven't tried to run this but what happens now if it's run?

```
class IntrospectionDisabledException(Exception):
    def __init__(self, message="Introspection is disabled."):
        self.message = message
        super().__init__(self.message)
```

Reply:
Locally, I am allowing introspection since we need to look at schema. But when you are in Prod Mode (Controlled via Django-debug setting), it should disallow

Reply:
It will throw error message